src/main/java/org/eln2/mc/common/cell/types/12VBatteryCell.kt

```diff
@@ -0,0 +1,75 @@
+package org.eln2.mc.common.cell.types
+
+import net.minecraft.core.BlockPos
+import org.ageseries.libage.sim.electrical.mna.component.Resistor
+import org.ageseries.libage.sim.electrical.mna.component.VoltageSource
+import org.eln2.mc.common.cell.CellBase
+import org.eln2.mc.common.cell.ComponentInfo
+import org.eln2.mc.extensions.ComponentExtensions.connectToPinOf
+import org.eln2.mc.utility.UnitType
+import org.eln2.mc.utility.ValueText
+
+class `12VBatteryCell`(pos : BlockPos) : CellBase(pos) {
+
+    /*
+    *   V -> local voltage source.
+    *   C -> remote components.
+    *
+    *   C V C
+    */
+
+    private lateinit var source : VoltageSource
+
+    override fun clearForRebuild() {
```

Comment:
This is a problem in a few other places as well, but we should restore the potentials and other information from the save file when rebuilding.

As another example: the voltage source doesn't persist changes to voltage across world loads.